pygmt/base_plotting.py

```diff
@@ -420,7 +424,9 @@ def grdview(self, grid, **kwargs):
                     drapegrid = kwargs["G"]
                     if data_kind(drapegrid) in ("file", "grid"):
                         if data_kind(drapegrid) == "grid":
-                            drape_context = lib.virtualfile_from_grid(drapegrid)
+                            drape_context = lib.virtualfile_from_grid(
+                                drapegrid, coord_sys, in_reg
```

Comment:
@seisman, your idea of using xarray attributes would actually make sense here in `grdview`, as we would otherwise need two `in_reg`s to deal with the drapegrid and reliefgrid separately.

Reply:
`grdmath` may also take more than one grid.